index.js

```diff
@@ -374,24 +376,40 @@ async function runTests(handle) {
 
     const testRunResponse = await SF.createTestRun(testParams);
     const testRunId = testRunResponse.data;
+    console.log(testRunId);
+
     let testRun = { status: "started" };
     const pollingDelay = 2000;
+    const spinner = new Spinner("%s Running tests..");
+    spinner.setSpinnerString("⠋⠙⠹⠸⠼⠴⠦⠧⠇⠏");
+    spinner.start();
 
     while (testRun.status === "started") {
-      await new Promise((resolve) => setTimeout(resolve, pollingDelay));
+      await new Promise((resolve) => {
+        setTimeout(resolve, pollingDelay);
+      });
       const response = await SF.fetchTestRun(testRunId);
       testRun = response.data;
     }
 
+    spinner.stop();
+    readline.clearLine(process.stdout);
+    readline.cursorTo(process.stdout, 0);
+
     if (testRun.status !== "completed") {
```

Comment:
Should we look at more precise codes ? 
Instead of `!== "completed"`, `=== "test_error"` ?
Do we have a list of all the possible status?

Reply:
Would need to ask ENG for a list cause not sure if there's a specific error code